(35, 'Unknown SSL protocol error in connection to api.github.com:443 ')
(28, 'Operation timed out after 80004 milliseconds with 140291 out of 549345 bytes received')
(28, 'Operation timed out after 80000 milliseconds with 379087 out of 437479 bytes received')
4095 crawler success
2847808 crawler success
12744739 crawler success
Traceback (most recent call last):
  File "issues_get.py", line 92, in <module>
    issues_get()
  File "issues_get.py", line 32, in issues_get
    repo_id=repo.split(" ")[0]
AttributeError: 'NoneType' object has no attribute 'split'
